packages/backend/server/src/__tests__/models/doc.spec.ts

```diff
@@ -115,8 +119,8 @@ test('should find updates by spaceId and docId', async t => {
   ]);
   const foundUpdates = await t.context.doc.findUpdates(workspace.id, docId);
   t.is(foundUpdates.length, 2);
-  t.deepEqual(foundUpdates[0].blob, Buffer.from('blob1'));
-  t.deepEqual(foundUpdates[1].blob, Buffer.from('blob2'));
+  t.deepEqual(foundUpdates[0].blob, textToUint8Array('blob1'));
```

Comment:
we can update all buffer usage to uint8array like `new Uint8Array([1, 2, 3])`. In most of the test cases, the content of binary is not used